src/main.rs

```diff
@@ -1,8 +1,11 @@
+use chrono::{DateTime, Duration, FixedOffset, ParseError};
```

Comment:
So, you basically suggest to completely revert this change? :wrench:

Reply:
Yes 😄

Reply:
:wrench: :heavy_check_mark: